# Fields
std_msgs/Header header

nao_landmark_detection/Info[] info
